app/views/cities/index.js.erb

```diff
@@ -1,9 +1,9 @@
 // Find the city select box
 var volunteer_city = document.getElementById("volunteer_city");
-let hospital_city = document.getElementById('hospital_city');
-let partner_city = document.getElementById('partner_city');
+var hospital_city = document.getElementById('hospital_city');
```

Comment:
Do the `let` statements need to change? I understand why `const` might be a problem, but not `let`. `let` allows variables to be re-assigned.